scripts/local_testnet/stop_local_testnet.sh

```diff
@@ -8,3 +8,7 @@ source ./vars.env
 PID_FILE=$TESTNET_DIR/PIDS.pid
 ./kill_processes.sh $PID_FILE
 rm -f $PID_FILE
+
+# Not all geth processes are killed when cleaning the PID_FILE. See:
+# https://github.com/sigp/lighthouse/issues/5382
+killall -HUP geth || true
```

Comment:
This could kill another geth running in the background for non-testing purposes

Reply:
Yeah, you're right. This is kind of an ugly hack in hindsight 😅. 

I'll try to look deeper into this issue later this week.